test/integration/graql/query/GraqlDefineIT.java

```diff
@@ -407,104 +143,14 @@ public void whenDefiningAThing_Throw() {
-        exception.expectMessage(anyOf(
-                is(GraqlSemanticException.defineUnsupportedProperty(Graql.Token.Property.HAS.toString()).getMessage()),
-                is(GraqlSemanticException.defineUnsupportedProperty(Graql.Token.Property.VALUE.toString()).getMessage())
-        ));
-
-        tx.execute(Graql.define(var().id(id.getValue()).has("title", "Bob")));
-    }
-
-    @Test @Ignore
-    public void whenSpecifyingLabelOfAnExistingConcept_LabelIsChanged() {
-        tx.putEntityType("a-new-type");
-
-        EntityType type = tx.getEntityType("a-new-type");
-        Label newLabel = Label.of("a-new-new-type");
-
-        // TODO: figure out how this was possible in the first place
-        //       how could we modify the label of a type by its ID????
-        tx.execute(Graql.define(type(newLabel.getValue()).id(type.id().getValue())));
-
-        assertEquals(newLabel, type.label());
-    }
-
     @Test
     public void whenCallingToStringOfDefineQuery_ReturnCorrectRepresentation() {
         String queryString = "define my-entity sub entity;";
         GraqlDefine defineQuery = Graql.parse(queryString).asDefine();
         assertEquals(queryString, defineQuery.toString());
     }
 
-    @Test
-    public void whenDefiningARelation_SubRoleDeclarationsCanBeSkipped() {
-        tx.execute(Graql.define(type("marriage").sub(Graql.Token.Type.RELATION).relates("husband").relates("wife")));
-
-        RelationType marriage = tx.getRelationType("marriage");
-        Role husband = tx.getRole("husband");
-        Role wife = tx.getRole("wife");
-        assertThat(marriage.roles().toArray(), arrayContainingInAnyOrder(husband, wife));
-    }
-
-    @Test
-    public void whenDefiningARelation_SubRoleCasUseAs() {
-        tx.execute(Graql.define(type("parentship").sub(Graql.Token.Type.RELATION)
-                          .relates("parent")
-                          .relates("child")));
-        tx.execute(Graql.define(type("fatherhood").sub(Graql.Token.Type.RELATION)
-                          .relates("father", "parent")
-                          .relates("son", "child")));
-
-        RelationType marriage = tx.getRelationType("fatherhood");
-        Role father = tx.getRole("father");
-        Role son = tx.getRole("son");
-        assertThat(marriage.roles().toArray(), arrayContainingInAnyOrder(father, son));
-        assertEquals(tx.getRole("parent"), father.sup());
-        assertEquals(tx.getRole("child"), son.sup());
-    }
-
-    @Test
-    public void whenDefiningARule_SubRuleDeclarationsCanBeSkipped() {
-        Pattern when = Graql.parsePattern("$x isa entity;");
-        Pattern then = Graql.parsePattern("$x isa entity;");
-        Statement vars = type("my-rule").when(when).then(then);
-        tx.execute(Graql.define(vars));
-
-        assertNotNull(tx.getRule("my-rule"));
-    }
-
-    @Test
-    public void whenDefiningARelation_SubRoleDeclarationsCanBeSkipped_EvenWhenRoleInReferredToInOtherContexts() {
```

Comment:
we got a test like this?

Reply:
It tests that we don't need `sub role` to define a role. Given that we're phasing out `sub role`, the test has become redundant.